lib/sqlalchemy/sql/operators.py

```diff
@@ -670,7 +674,11 @@ def isnot(self, other):
         .. seealso:: :meth:`.ColumnOperators.is_`
 
         """
-        return self.operate(isnot, other)
+        return self.operate(is_not, other)
+
+    def isnot(self, other):
+        # SqlAlchemy 1.4 deprecates legacy terms for new standardizations (see Issue#5429)
+        return self.is_not(other)
```

Comment:
Since at the moment we are not deprecating them with a warning, I would just do `isnot = is_not` so we avoid a function call. Same also for `notin_`

Reply:
100% agree. I tried to use that pattern elsewhere, not sure how this one slipped through.